### PR TITLE
Admin Guide Fix /  master-fingerprint.html

### PR DESCRIPTION
**Pull request by 0rnela**
https://github.com/SUSE/spacewalk/issues/24348

### CODE REVIEW

File: modules/administration/pages/master-fingerprint.adoc [hunk 1]
```diff
@@ -1,11 +1,21 @@
 [[security-master-fingerprint]]
 = Set up a Client to Master Validation Fingerprint
 
-In highly secure network configurations you may wish to ensure your Salt clients are connecting a specific master.
-To set up validation from client to master enter the master's fingerprint within the [path]``/etc/salt/minion`` configuration file, see the following procedure:
 
-.Procedure: Adding Master's Fingerprint to Client
+In highly secure network configurations you may wish to ensure your {salt} clients are connecting a specific master.
+To set up validatfrom client to master start by entering the master's fingerprint within the configuration file:
```
Comment (keichwa) [inline modules/administration/pages/master-fingerprint.adoc:6]:
```suggestion
To set up validation from client to master start by entering the master's fingerprint within the configuration file:
```

**Review by keichwa — CHANGES_REQUESTED** [review]
just a typo

File: modules/administration/pages/master-fingerprint.adoc [hunk 2]
```diff
@@ -1,11 +1,21 @@
...
 
-In highly secure network configurations you may wish to ensure your Salt clients are connecting a specific master.
-To set up validation from client to master enter the master's fingerprint within the [path]``/etc/salt/minion`` configuration file, see the following procedure:
 
-.Procedure: Adding Master's Fingerprint to Client
+In highly secure network configurations you may wish to ensure your {salt} clients are connecting a specific master.
+To set up validation from client to master start by entering the master's fingerprint within the configuration file:
+
+* [path]``/etc/salt/minion.d/custom.conf`` in cases of using classic {salt} minion in your client, or
+* [path]``/etc/venv-salt-minion/minion.d/custom.conf`` in case of using {salt} Bundle in your client
```
Comment (meaksh) [inline modules/administration/pages/master-fingerprint.adoc:9]: These mentioned `custom.conf` files are not existing by default on a system, the user should have to create.

So I would rephrase to:

```suggestion
In highly secure network configurations you may wish to ensure your {salt} clients are connecting a specific master.
To set up validation from client to master start by entering the master's fingerprint within a {salt} minion configuration file:

* [path]``/etc/salt/minion.d/custom.conf`` in cases of using classic {salt} minion in your client, or
* [path]``/etc/venv-salt-minion/minion.d/custom.conf`` in case of using {salt} Bundle in your client
```

File: modules/administration/pages/master-fingerprint.adoc [hunk 3]
```diff
@@ -14,19 +24,23 @@ salt-key -F master
 ----
 +
 
-On your client, open the [path]``/etc/salt/minion`` configuration file.
+On your client, open the [path]``/etc/salt/minion`` or [path]``/etc/venv-salt-minion/minion.d/custom.conf`` configuration file.
```
Comment (meaksh) [inline modules/administration/pages/master-fingerprint.adoc:27]: For consistency, we probably should mention `/etc/salt/minion.d/custom.conf` instead of `/etc/salt/minion`:

```suggestion
On your client, open the [path]``/etc/salt/minion.d/custom.conf`` or [path]``/etc/venv-salt-minion/minion.d/custom.conf`` configuration file.
```

File: modules/administration/pages/master-fingerprint.adoc [hunk 4]
```diff
@@ -14,19 +24,23 @@ salt-key -F master
...
 ----
 
-. Restart the salt-minion service:
+. Restart the salt-minion or venv-salt-minion service.
+  For venv-salt-minion, run:
 +
 
 ----
-# systemctl restart salt-minion
+systemctl restart vent-salt-minion
```
Comment (meaksh) [inline modules/administration/pages/master-fingerprint.adoc:40]: Typo:

```suggestion
systemctl restart venv-salt-minion
```

File: modules/administration/pages/master-fingerprint.adoc [hunk 5]
```diff
@@ -14,19 +24,23 @@ salt-key -F master
...
 ----
 
+In case of using {salt} Bundle, instead of [literal]``salt-minion`` use [literal]``venv-salt-minion``.
```
Comment (meaksh) [inline modules/administration/pages/master-fingerprint.adoc:43]: This statement should be the opposite, isn't it? As we already provided the example for `venv-salt-minion`:

Or maybe we should keep this statement here, but then provide the above example of restarting the service with the classic Salt example: `systemctl restart salt-minion`

**Review by meaksh — CHANGES_REQUESTED** [review]
Thanks for the PR! I added some comments I think we should address

File: modules/administration/pages/master-fingerprint.adoc [hunk 6]
```diff
@@ -14,19 +24,29 @@ salt-key -F master
 ----
 +
 
-On your client, open the [path]``/etc/salt/minion`` configuration file.
+On your client, open the [path]``/etc/salt/minion.d/custom.conf`` or [path]``/etc/venv-salt-minion/minion.d/custom.conf`` configuration file.
 Uncomment the following line and enter the master's fingerprint replacing the example fingerprint:
```
Comment (meaksh) [inline modules/administration/pages/master-fingerprint.adoc:28]: This we should also change, as the file is created by the user then it is empty, so the user cannot "uncomment the following line". We should ask them to add the file

**Review by meaksh — CHANGES_REQUESTED** [review]
Thanks for the changes! Looks much better now but I just noticed one more thing we should change :)
